Sources/BraintreeThreeDSecure/Public/BraintreeThreeDSecure/BTThreeDSecureRequest.h

```diff
@@ -45,6 +59,13 @@ typedef NS_ENUM(NSInteger, BTThreeDSecureVersion){
  */
 @property (nonatomic, copy) NSDecimalNumber *amount;
 
+/**
+ Optional. The account type selected by the cardholder
+
+ @note Some cards can be processed using either a credit or debit account and cardholders have the option to choose which account to use.
+ */
+@property (nonatomic, assign) BTThreeDSecureAccountType accountType;
```

Comment:
I think if we're going to make this an enum, we should also consider doing the same for the `shippingMethod` property on `BTThreeDSecureRequest`.

Reply:
Yeah, I agree. Let's do that as a separate PR, though.